spec/dummy/tmp/test_track_schema.yml

```diff
@@ -0,0 +1,13 @@
+---
+identifier_types:
+- a
+- b
+- c
+splits:
+  balance_unit:
+    dollar: 50
+    pound: 25
+    doge: 25
+  blue_button:
+    'false': 50
+    'true': 50
```

Comment:
oh hrm, browsed right over this.  yea maybe gitignore everything but the .keep file.